corehq/apps/hqwebapp/templates/hqwebapp/base.html

```diff
@@ -18,6 +18,9 @@
             {% if not title_context_block %}CommCare HQ{% endif %}
         </title>
 
+        {% captureas requirejs_main_block %}{% block requirejs_main %}{% endblock requirejs_main %}{% endcaptureas %}
+        {% set_requirejs_main request requirejs_main_block %}
```

Comment:
...following from [suggested movement of `set_requirejs_main`](https://github.com/dimagi/commcare-hq/pull/17824/commits/c34703accfb1f66547ada7faa10bb7fc52e61a44#diff-72a4eb316e6aad7b47cf16dfd80d114c), I think these two lines could be removed entirely.